Polygon_mesh_processing/examples/Polygon_mesh_processing/corefinement_mesh_union_progress.cpp

```diff
@@ -0,0 +1,217 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <CGAL/Surface_mesh.h>
+
+#include <CGAL/Polygon_mesh_processing/corefinement.h>
+#include <CGAL/Polygon_mesh_processing/IO/polygon_mesh_io.h>
+
+#include <fstream>
+
+typedef CGAL::Exact_predicates_inexact_constructions_kernel   K;
+typedef CGAL::Surface_mesh<K::Point_3>                        Mesh;
+
+namespace PMP = CGAL::Polygon_mesh_processing;
+
+struct Visitor_rep{
+
+  Visitor_rep(double normalize = 4)
+    : normalize(normalize)
+  {
+    t.start();
+  }
+
+  void progress_filtering_intersections(double d)
+   {
+    d /= normalize;
+    total += d;
+    if(total > bound){
+      std::cout << std::setprecision(3) << total*100 << " %   in " << std::setprecision(5) << t.time() << " sec." << std::endl;
+      bound += 0.1;
+    }
+  }
+
+  void start_triangulating_faces(std::size_t tf)
+  {
+    tfaces = tf;
+    bound_faces = tf/10;
+  }
+
+  void face_triangulation(std::size_t i)
+  {
+    if(i> bound_faces){
+      std::cout << double(i)/double(tfaces) * 100  << " %" << std::endl;
+      bound_faces += tfaces/10;
+    }
+  }
+
+  void start_coplanar_faces(std::size_t tc)
+  {
+    std::cout << "Visitor::start_coplanar_faces() at " << t.time() << " sec." << std::endl;
+    tcoplanar= tc;
+    count_coplanar = 0;
+    bound_coplanar = tcoplanar/10;
+  }
+
+  void intersection_of_coplanar_faces_step()
```

Comment:
Detail, but the concept asks for const functions

Reply:
@sloriot is that only a detail?

Reply:
that was a copy/paste error in the concept from the default visitor.